site/docs/v1/tech/apis/_two-factor-disable-request-body.adoc

```diff
@@ -0,0 +1,16 @@
+==== Request Body
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [optional]#Optional#::
+The Id of the application. If valid, the email template configured in the Application settings will be used, if present. If not present, the email template configured in the Tenant settings will be used. In either case, the corresponding Application object will be available to the template.
+To learn more about overriding email templates, read our link:/docs/v1/tech/guides/configuring-application-specific-email-templates[Application Specific Email Templates] guide.
+
+[field]#code# [type]#[String]# [required]#Required#::
+A valid Multi-Factor verification code. This value should be provided by the User to verify they are able to produce codes.
```

Comment:
That description for `code` appears 4 times on the Two Factor Page so I would have to update it. Here's my new description:


_A valid Two-Factor verification code. The User will obtain this code by either a Time-based one-time password (TOTP), Email, or SMS to prove they have access to codes._

The `applicationId` can be included in the Request Body - this is what I originally tested and found doing an audit. The request parameters are URL parameters. Our code is pulling the applicationId off of the request body. I will re-test to confirm again to make sure I am getting it right because it has been a few weeks.